File: pset1 - C/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main (void)
{
    // 25c, 10c, 5c, 1c
    // get_value
    float dollars;
    do {
        dollars = get_float("Dollars: ");
    }
    while (dollars < round (0.01));

    int cents = round (dollars * 100); // Em cents
    int coins = 0; // Contar moedas

    printf("You have = %i C \n", cents);

    // Repetir até que cents seja maior ou igual a 0
    while (cents > 0) {
        if (cents >= 25) {
            cents -=25;
            coins ++;
        } else if (cents >= 10) {
            cents -= 10;
            coins ++;
        } else if (cents >= 5) {
            cents -= 5;
            coins ++;
        } else {
            cents -= 1;
            coins ++;
        }
    }

    printf ("%d\n", coins);
}

File: pset6 - Python/cash/cash.py
from cs50 import get_float

while True:
    cents = get_float("Change: ")
    if cents > 0:
        break

cents = round(cents*100)

count = 0

# Moedas de 25
while cents >= 25:
    cents = cents - 25
    count += 1

# Moedas de 10
while cents >= 10:
    cents = cents - 10
    count += 1

# Moedas de 5
while cents >= 5:
    cents = cents - 5
    count += 1

# Moedas de 1
while cents >= 1:
    cents = cents - 1
    count += 1

print("Total coins: ", count)
